APNGKit/APNGImageView.swift

```diff
@@ -24,8 +24,16 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
-import UIKit
-
+#if os(OSX)
+    import Cocoa
+    public typealias APNG_View = NSView
```

Comment:
I am not sure an `_` in type name is good or not. Maybe I prefer to use `APNGView` instead.